src/uct/ib/rdmacm/rdmacm_cm.c

```diff
@@ -88,7 +89,8 @@ static void uct_rdmacm_cm_handle_event_addr_resolved(struct rdma_cm_event *event
               uct_rdmacm_cm_ep_str(cep, ep_str, UCT_RDMACM_EP_STRING_LEN),
               event->id);
 
-    if (rdma_resolve_route(event->id, 1000 /* TODO */)) {
+    if (rdma_resolve_route(event->id, UCS_MSEC_PER_SEC *
+                           cm->super.config.resolve_timeout)) {
```

Comment:
static inline int uct_rdmacm_get_timeout(cm) 
{
  return  UCS_MSEC_PER_SEC * cm->super.config.resolve_timeout
}

reuse in uct_rdamcm_cm_ep_client_init